NEWS.rst

```diff
@@ -72,6 +72,7 @@ possible, especially the following contributors:
 - Michiel de Hoon
 - Peter Cock
 - Sergio Valqui
+- Vini Salazar (first contribution)
```

Comment:
That's the names for Biopython 1.78 (already done), you need to add yourself at about the current line 29 instead.

Reply:
Thanks, fixed it. I hadn't added it previously because wasn't sure where to add.